setup.py

```diff
@@ -5,10 +5,20 @@
 
 INSTALL_REQUIREMENTS = [
     "Django>=1.11,<2.0",
-    "django-cms>=3.5.0",
+    'django_cms',
```

Comment:
This package is dependant on djamgocms versioning which is Django CMS 4.

Reply:
Yes, if you look at line 44 the dependency link is to version 4 of django cms.